docs/tutorial_basics.rst

```diff
@@ -252,7 +255,8 @@ This is exactly what we want to do here, and we know that the test sources are n
 Hence, we move the ``sourcepath`` assignment into a pre-compile hook.
 
 .. literalinclude:: ../tutorials/basics/hello/hello2.py
-   :lines: 17-19
+   :start-after: # rdmdocstart: set_sourcepath
+   :end-before: # rdmdocend: set_sourcepath
```

Comment:
```suggestion
   :start-after: # rfmdocstart: set_sourcepath
   :end-before: # rfmdocend: set_sourcepath
```
I assume this is by mistake. If not, it's really not important, it can stay as it is.

Reply:
Yeah, thanks for noticing. `f` is next to `d` on the keyboard :)